org.eclipse.lsp4xml/src/main/java/org/eclipse/lsp4xml/settings/XMLTelemetrySettings.java

```diff
@@ -8,15 +8,21 @@
 *     Red Hat Inc. - initial API and implementation
 *******************************************************************************/
 
-package org.eclipse.lsp4xml.utils;
+package org.eclipse.lsp4xml.settings;
 
 /**
- * OSUtils
+ * Telemetry settings
  */
-public class OSUtils {
-	
-	public static final boolean isWindows = System.getProperty("os.name").toLowerCase().indexOf("win") >= 0;
-	public static String SLASH = isWindows ? "\\" : "/";
+public class XMLTelemetrySettings {
 
-	
-}
\ No newline at end of file
+	private boolean enabled = true;
```

Comment:
should be off by default, clients should explicitly enable it

Reply:
Good catch! It was a compy from XMLSymbolSettings. Copy/paste generate so many errors :)